load-testing/initial-load-testing.md

```diff
@@ -0,0 +1,89 @@
+# Initial load testing
+
+## Files used
+
+A 3Gb, 40000 second wav file, created using ffmpeg.
+
+2 2.8Gb bmp files created using ImageMagick's convert tool
+
+2000 PDFs created using ImageMagick's convert tool
+
+10000 PDFs created using ImageMagick's convert tool
+
+These files have been uploaded to the tdr-upload-test-data bucket in the Sandbox environment if anyone wants to use them.
+
+## Running the tests
+
+I carried out the upload on a t3.medium EC2 instance with 2 CPU cores and 4GB RAM. Our first users will have a similar CPU with 8GB of RAM so the test machine is lower spec.
+
+I did have to make some changes to various timeouts otherwise none of these would have worked.
+* The akka request timeout on the API, which defaults to 20s, was switched off.
+* The akka server timeout on the API, which defaults to 60s, was switched off.
+* The load balancer idle timout, which was set at 60s, was increased to 10 minutes.
```

Comment:
Do you have a note of the limits without setting these timeouts? Increasing timeouts is something I'd be very cautious about doing in production.

Reply:
I switched the timeouts off for the tests just to save me having to increase them every time I guessed wrong. I mentioned in the potential fixes about increasing the timeouts but I've moved it to the longer term fixes as there are other things we can do like pagination, auto-scaling, etc. which will reduce the time needed for the queries rather than increasing the the timeouts. I think this is what you meant in the comment below about the timeouts as well.